### ISSUE TITLE
Triggering rule align_parameters when declaring base fields in itab VALUE syntax

### ISSUE THREAD

**Issue by Fxlipe115** [abaplint/abaplint#2648]
The code below uses syntax for declaring columns with the same value on multiple lines when declaring an internal table. As can be seen, the parameters with common values do not align with those with different values in each line.

![image](https://user-images.githubusercontent.com/18754974/185944263-16984cdb-be29-4b81-832f-6f0835e8bf1a.png)

My team uses this style because the parameters with common values are in a different identation. It also helps us quickly identify which fields have default values and which do not, mainly in unit tests. The problem is this triggers the align_parameters rule in abaplint.

Since there is no official guideline for this specific case in [CleanABAP](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#align-parameters), and since this rule applies to a variety of constructs in the language, it would be nice to be able to choose in which constructs we want this rule to apply. For example:
![image](https://user-images.githubusercontent.com/18754974/185948763-c2fd01ad-d56d-4b5a-9ec0-413bfcd96cfb.png)

**Comment by larshp** [comment]
I think it would be okay to change the logic to consider following okay,

```abap
DATA(sdf) = VALUE type(
  common_val     = 2
  another_common = 5
  ( row_value = 4
    value_foo = 5 ) ).
```

**Comment by larshp** [comment]
@Fxlipe115 try again, 2.92.2 published

**Comment by Fxlipe115** [comment]
It is working perfectly. Thank you very much.
